sphinx-doc/components.rst

```diff
@@ -11,7 +11,7 @@ Components can be compiled and installed as **built-in** components or as
 **external** components. The build process compiles HOOMD-blue and all built-in
 components together, requiring one ``cmake`` and ``make``. External components
 compile and link against a HOOMD-blue installation with a separate invocation of
-``cmake`` and ``make install``. A properly configured component may be compiled
+``cmake`` and ``cmake --install``. A properly configured component may be compiled
```

Comment:
Is this to ensure whatever build system selected at configuration is used?

Reply:
Yes. `cmake --build` and `cmake --install` are cross-platform and independent of the build tool. CMake forwards these commands onto the build tool selected at the configure step. I removed references to `make install` since they are specific to the make tool. Users that are familiar with their tools are of course welcome to use tool specific commands.

Reply:
I reworded this section to match the new suggested general syntax in `BUILDING.rst`